src/VisualStudioExtension/src/Tanzu.Toolkit.VisualStudioExtension.csproj

```diff
@@ -156,6 +156,14 @@
     <Resource Include="Views\Resources\Pause.png" />
     <Resource Include="Views\Resources\RunPaused.png" />
     <Resource Include="Views\Resources\Stop.png" />
+    <Content Include="Resources\CFSsh.dll">
```

Comment:
Should these references to `CFSsh.*` should be `CfSshWrapper.*`?

Reply:
Yeah they definitely should; in the moment I thought I was saving effort by not rebuilding/replacing the binaries here since this whole exe design is still in flux & they're likely to change soon anyway. In retrospect though, that meant that the debug adapter rename in `FileService` (from `CFSsh.exe` to `CfSshWrapper`) was a breaking change.

Will fix in a follow-up commit